.github/workflows/stale.yml

```
@@ -0,0 +1,93 @@
+name: Handle stale issues
+on:
+  schedule:
+  - cron: "30 2 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+    permissions:
+      issues: write  # for actions/stale to close stale issues
+      pull-requests: write  # for actions/stale to close stale PRs
+    if: github.repository_owner == 'osgeo'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v8
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: >
+          The PROJ project highly values your contribution and would love to see
+          this work merged!
+          Unfortunately this PR has not had any activity in the last 14 days and
```

```suggestion
          Unfortunately this PR has not had any activity in the last two months and
```
because `days-before-pr-stale: 60`

---

Thanks. I knew I would have missed at least one of them!